src/export.rs

```diff
@@ -119,12 +119,9 @@ pub fn export_system_to_csv_by_body(
         ))?;
 
         // here we remove the generated newline character from the csv library so that we can use writeln below.
-        let text = String::from_utf8(wtr.into_inner()?)?.replace("\n", "");
+        let text = String::from_utf8(wtr.into_inner()?)?.replace('\n', "");
 
-        let mut file = OpenOptions::new()
-            .write(true)
-            .append(true)
-            .open(&fullpath)?;
+        let mut file = OpenOptions::new().append(true).open(&fullpath)?;
```

Comment:
I'm curious what Clippy said about this 😄

Reply:
Exactly, append(true) implies write(true)